5.5/root/usr/bin/run-mysqld

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+export_vars=$(cgroup-limits); export $export_vars
```

Comment:
It's there in the usage of the script :) You just forgot to use set -o errexit. The reason it is two separate commands is so that if cgroup-limits fails for some reason, the whole script fails with it. What I do not understand is why this was called before the source and set lines bellow.